src/client/app/Sidebar/TopPerformers/TopPerformers.js

```diff
@@ -52,7 +52,9 @@ const getPerformerLinks = performer => {
 
 const formatPerformance = performanceValue => {
   const plusSign = performanceValue > 0 ? '+' : '';
-  return `${plusSign}${toFixNumberLength(performanceValue, 3)}%`;
+  return performanceValue !== null
```

Comment:
you can use lodash 'isNil' to check 'undefined' and null values. Also you can validate argument at the first line of function